### PR TITLE
BW-620 - Create Step 2: after Singup for first question

### PR DESCRIPTION
**Pull request by jitendraashutec**

### CODE REVIEW

File: App_Resources/Android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 	package="__PACKAGE__"
-	android:versionCode="${versionCode}"
-	android:versionName="${versionName}">
+	android:versionCode="28"
```
Comment (milansar) [inline App_Resources/Android/src/main/AndroidManifest.xml]: why this is changed?

**Comment by milansar** [comment]
@anihalaney , PR is ready for your review

File: functions/controllers/user.controller.ts [hunk 2]
```diff
@@ -231,4 +231,20 @@ export class UserController {
             Utils.sendError(res, error);
         }
     }
+
+    static async addBitesFirstQuestion(req, res) {
```
Comment (anihalaney) [inline functions/controllers/user.controller.ts]: typo. should be addBitsFirstQuestion no "e" in Bits

File: projects/shared-library/src/lib/shared/mobile/component/first-question/first-question.component.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        if (appSettings) {
+          this.applicationSettings = appSettings[0];
+          this.cd.markForCheck();
+        }
+      }));
+
+    this.store.select(categoryDictionary).pipe(map(categories => {
+      this.categoryDictionary = categories;
+    }),
+      flatMap(() => this.store.select(appState.coreState).pipe(select(s => s.firstQuestion)))).subscribe((question: Question) => {
```
Comment (anihalaney) [inline projects/shared-library/src/lib/shared/mobile/component/first-question/first-question.component.ts]: missing this.subscriptions.push(

File: projects/shared-library/src/lib/shared/model/constants.ts [hunk 4]
```diff
@@ -373,6 +374,7 @@ export enum RoutesConstants {
     LAT_LONG = 'latLong',
     ADDRESS_SUGGESTION = 'addressSuggestion',
     UPDATE_APP_VERSION = 'updateAppVersion',
+    ADD_BITES_FIRST_QUESTION = 'add-bits-first-question',
```
Comment (anihalaney) [inline projects/shared-library/src/lib/shared/model/constants.ts]: BITS

**Pull request merged**